code/web/src/modules/user/Profile.js

```diff
@@ -31,16 +35,42 @@ const Profile = (props) => (
     </Grid>
 
```

Comment:
I ideally see down the road working in a turnary here, and having a boolean value of "isEditing" so that we will either conditionally render the current user info as is bellow, or will render a form that allows a user to change their description, image, and shipping adress.